__tests__/clientTests/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import App from '../../client/src/components/App.jsx';
+
+describe('does it render', () => {
```

Comment:
instead of asking vaguely, be more descriptive of what the test should do. Example: Renders shallow App component.

Reply:
Changed! Thank you for the feedback